### ISSUE TITLE
Error report for keycombiner 0.3.0

### ISSUE THREAD

**Issue by HortonsHelpers** [tkainrad/keycombiner#24]
Error:
```TypeError: Error processing argument at index 0, conversion failure from 
    at C:\Users\horto\AppData\Local\Programs\keycombiner\resources\app.asar\webpack:\src\main\index.js:729:20
    at FSReqCallback.oncomplete (fs.js:160:23)
```
OS: Windows NT 10.0.19041

The app requested I report this...

I am also having trouble using the shortcut key to pull up the quick reference guide. It doesn't activate even when the program is open. 

And lastly, a suggestion: Have the quick reference with an option to always be on top and automatically generate in the top right/left corner.

Thanks again!

**Comment by tkainrad** [comment]
Hi, thanks a lot for reporting this!

Someone recently had the same error: #22  
Their problem was that multiple KeyCombiner instances were running in parallel. Could it be the same in your case? KeyCombiner 0.3.1 will solve this problem by only allowing one instance to be run. It will be released within the next 3 weeks.

**Regarding your other issues/requests:** 

> I am also having trouble using the shortcut key to pull up the quick reference guide. It doesn't activate even when the program is open.

The instant lookup just never shows up? Can you try to change the lookup shortcut? (*File->Set Lookup Shortcut*) Maybe another software already listens on the same shortcut binding.

> And lastly, a suggestion: Have the quick reference with an option to always be on top and automatically generate in the top right/left corner.

I am sorry, I am not sure what you mean. You can already trigger the instant lookup also from the KeyCombiner icon in the notification area in the Windows taskbar. However, I suppose that's now what you mean.

By always being on the top and automatically generate, do you mean that it really should always be shown? But will this not be very annoying when it is overlapping other windows? I think you would still have to press the lookup shortcut to focus on the lookup's search field.

**Comment by tkainrad** [comment]
Closing this now as I think it was likely resolved by #23. Please feel free to submit it again if you still experience this problem.

**Comment by tkainrad** [comment]
Hi again @HortonsHelpers, quick update. I finally found the actual root of your problem:

It means that you set an empty string as the instant lookup trigger shortcut. See #55 for more details.
